src/cli/commands/config.js

```diff
@@ -16,8 +16,15 @@ export const {run, setFlags} = buildSubCommands('config', {
       return false;
     }
 
-    const [key, val = true] = args;
+    const [key, val] = ((key, val = true) => {
+      if (typeof val === 'string' && (/^(true|false)$/i).test(val)) {
```

Comment:
Can you change this to `val === 'false' || val === 'true', it's not super obvious what this does on first glance.